File: staGledas.Model/Models/KlubLajkovi.cs
using System;

namespace staGledas.Model.Models
{
    public class KlubLajkovi
    {
        public int Id { get; set; }
        public int ObjavaId { get; set; }
        public int KorisnikId { get; set; }
        public DateTime DatumLajka { get; set; }
        public virtual KlubObjave? Objava { get; set; }
        public virtual Korisnici? Korisnik { get; set; }
    }
}

File: staGledas-UI/stagledas_mobile/lib/models/klub_lajkovi.dart
import 'package:json_annotation/json_annotation.dart';
import 'korisnik.dart';

part 'klub_lajkovi.g.dart';

@JsonSerializable()
class KlubLajkovi {
  int? id;
  int? objavaId;
  int? korisnikId;
  DateTime? datumLajka;
  Korisnik? korisnik;

  KlubLajkovi({
    this.id,
    this.objavaId,
    this.korisnikId,
    this.datumLajka,
    this.korisnik,
  });

  factory KlubLajkovi.fromJson(Map<String, dynamic> json) =>
      _$KlubLajkoviFromJson(json);

  Map<String, dynamic> toJson() => _$KlubLajkoviToJson(this);
}
